### PR TITLE
DOCSP-34600 Mention that exp field is ignored for custom JWT

### PR DESCRIPTION
**Pull request by lindseymoore**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-34600
Staged changes:

- [Custom JWT Authentication](https://preview-mongodblindseymoore.gatsbyjs.io/atlas-app-services/DOCSP-34600/authentication/custom-jwt/)

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [ ] Describe your PR's changes in the Release Notes section
- [ ] Create a Jira ticket for corresponding docs-realm updates, if any

### Release Notes

- Authenticate and Manage Users/Authentication Providers/Custom JWT: Note that App services specifies a 30 minute expiry for the access token even if expiry is defined by the Custom JWT's ``exp`` key. 

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/authentication/custom-jwt.txt [hunk 1]
```diff
@@ -4,6 +4,10 @@
 Custom JWT Authentication
 =========================
 
+.. facet::
```
Comment (cbullinger) [inline source/authentication/custom-jwt.txt:7]: nice (I keep forgetting to do this 😅 )

File: source/authentication/custom-jwt.txt [hunk 2]
```diff
@@ -71,6 +75,12 @@ JWT authentication with App Services follows these general steps:
    #. App Services checks if the signature in the JWK matches the JWT's 
       signature. If so, the user is authenticated.
 
+.. note:: Access Token Expires after 30 Minutes
```
Comment (cbullinger) [inline source/authentication/custom-jwt.txt:78]:
```suggestion
.. important:: Access Token Always Expires after 30 Minutes
```

File: source/authentication/custom-jwt.txt [hunk 3]
```diff
@@ -71,6 +75,12 @@ JWT authentication with App Services follows these general steps:
...
+
+  App Services specifies a 30 minute access token expiry even if the
```
Comment (cbullinger) [inline source/authentication/custom-jwt.txt:80]:
```suggestion
  App Services always specifies a 30-minute access token expiry even if the
```

File: source/authentication/custom-jwt.txt [hunk 4]
```diff
@@ -71,6 +75,12 @@ JWT authentication with App Services follows these general steps:
...
+  custom JWT token specifies a different expiry through the ``exp`` key. 
+  App Services will check the custom JWT token ``exp`` to ensure that the token
+  is still valid before issuing the 30 minute expiry.
```
Comment (cbullinger) [inline source/authentication/custom-jwt.txt:83]:
```suggestion
  is still valid before issuing the 30-minute expiry. For more information on App Services access tokens, refer to :ref:`manage-user-sessions`.
```

**Review by cbullinger — APPROVED** [review]
some non-blocking suggestions to increase the visibility of this info since it's not expected behavior

**Review by chickenbug — APPROVED** [review]
LGTM 🎉

**Pull request merged**